### ISSUE TITLE
Install with --local flag

### ISSUE THREAD

**Issue by bquorning** [ruby/setup-ruby#344]
When I add `ruby/setup-ruby` configured with `bundler-cache: true` to a project with gems packaged in vendor/cache/*.gem files, the first install would be much faster (and secure) if I could use the `--local` flag on the `bundle install` command.

Is that currently possible, or might it be considered as a new feature?

**Comment by dentarg** [comment]
I don't think it is possible and it has been brought up before in https://github.com/ruby/setup-ruby/pull/152, maybe you can continue the discussion started there here

**Comment by bquorning** [comment]
My apologies, this is indeed a duplicate. I searched but didn’t find that issue yesterday. I will continue the discussion in the issue you mentioned.
